src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -0,0 +1,146 @@
+	
+	/**
+	 * Insert a new {@link MedicalInventory}.
+	 *
+	 * @param medicalInventory - the {@link MedicalInventory} to insert
+	 * @return the newly persisted {@link MedicalInventory} object.
+	 * @throws OHServiceException
+	 */
+	public MedicalInventory newMedicalInventory(MedicalInventory medicalinventory) throws OHServiceException {
+			return ioOperations.newMedicalInventory(medicalinventory);
+	}
+	
+	/**
+	 * Update an existing {@link MedicalInventory}.
+	 *
+	 * @param medicalInventory - the {@link MedicalInventory} to update
+	 * @return the updated {@link MedicalInventory} object.
+	 * @throws OHServiceException
+	 */
+	public MedicalInventory updateMedicalInventoryRow(MedicalInventory medicalInventory) throws OHServiceException {
+		return ioOperations.updateMedicalInventoryRow(medicalInventory);
+	}
+	
+	/**
+	 * Delete the specified {@link MedicalInventory}.
+	 * @param medicalInventory - the {@link MedicalInventory} to delete.
+	 * @throws OHServiceException if an error occurs during the medicalInventory deletion.
+	 */
+	public void deleteMedicalInventory(MedicalInventory medicalInventory) throws OHServiceException {
+		ioOperations.deleteMedicalInventory(medicalInventory);
+	}
+	
+	/**
+	 * Check if the reference number is already used
+	 * 
+	 * @param reference - the {@link MedicalInventory} reference
+	 * @return {@code true} if the code is already in use, {@code false} otherwise
+	 * @throws OHServiceException
+	 */
+	public boolean referenceExists(String reference) {
+		return ioOperations.referenceExists(reference);
+	}
+	
+	/**
+	 * Return a list of {@link MedicalInventory}s for passed params.
+	 *
+	 * @param status - the {@link MedicalInventory} status.
+	 * @param wardCode - the {@link Ward} code.
+	 * @return the list of {@link MedicalInventory}s. It could be {@code empty}
+	 * @throws OHServiceException
+	 */
+	public List<MedicalInventory> getMedicalInventoryByStatusAndWard(String status, String wardCode) throws OHException {
+		return ioOperations.getMedicalInventoryByStatusAndWard(status, wardCode);
+	}
+	
+	/**
+	 * Return a list {@link MedicalInventory}s for passed params.
+	 *
+	 * @param status - the {@link MedicalInventory} status.
+	 * @return the list of {@link MedicalInventory}s. It could be {@code empty}
+	 * @throws OHServiceException
+	 */
+	public List<MedicalInventory> getMedicalInventoryByStatus(String status) throws OHException {
+		return ioOperations.getMedicalInventoryByStatus(status);
+	}
+	
+	/**
+	 * Return a list of results {@link MedicalInventory}s for passed params.
+	 *
+	 * @return the list of {@link MedicalInventory}s. It could be {@code empty}
+	 * @throws OHServiceException
+	 */
+	public List<MedicalInventory> getMedicalInventory() throws OHException {
+		return ioOperations.getMedicalInventory();
+	}
+	
+	/**
+	 * Return a list of {@link MedicalInventory}s for passed params.
+	 * 
+	 * @param dateFrom - the lower date for the range
+	 * @param dateTo - the highest date for the range
```

Comment:
```suggestion
	 * @param dateTo - the highest date for the range.
```
Please be consistent.